src/code/home/copy_data.asm

```diff
@@ -184,39 +183,48 @@ CopyDataToBGMap::
     ld   b, a                                     ; $298B: $47
     dec  b                                        ; $298C: $05
     ; Loop (without incrementing the source address)
-    jr   nz, .copyColumnSingleValue               ; $298D: $20 $F5
+    jr   nz, .fillColumn                          ; $298D: $20 $F5
     inc  de                                       ; $298F: $13
     ret                                           ; $2990: $C9
 
-label_2991::
+; Copy data to VRAM map, with support for copying a full row or column,
+; special-cased for code executed during a room transition.
+;
+; See DC_* constants for possible options.
+;
+; Inputs:
+;   de: data copy source address
+;   hl: data copy destination address
+;   a:  data length (bits 0-6) and copy mode (bits 7-8)
+DrawCommandToVRAMDuringRoomTransition::
```

Comment:
I didn't investigate the behavior differences between the normal `DrawCommandToVRAM` (outside of room transitions) and this `DrawCommandToVRAMDuringRoomTransition`. Doesn't look that much different, but I guess there are some subtleties.